### ISSUE TITLE
Installing 1.8 on Debian?

### ISSUE THREAD

**Issue by jhaupt** [shumatech/BOSSA#81]
No configure script in the tarball nor instructions for installing. 

Binaries would be equivalently good :)

**Comment by aswild** [comment]
You don't need a configure script, it's just a flat Makefile. Run "make" and then manually install the `bin/bossa*` binaries to `/usr/local/bin` or wherever else you want to install.

**Comment by thehans** [comment]
A list of dependencies would be nice and even if the instructions are simply "run make", its better to be explicit than to have everyone guessing what to do.  

I had to install `libwxgtk3.0-dev` for the makefile to run.  Not sure if there were other deps that I just happened to already have installed.

**Comment by aswild** [comment]
I think wxgtk and readline are the only dependencies, or at least that's all I have in my [Arch PKGBUILD](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=bossa-git)

**Comment by Nightbr** [comment]
Hey, I had to install wxgtk and realine like @aswild said:
`sudo apt install libreadline-dev libwxgtk3.0-dev`

It could be great to add this in some install documentation or juste compile some binaries for linux!

Thanks for your work!

**Comment by twitchyliquid64** [comment]
I've whipped up this script to automatically build 1.9.1 from source and install it.

https://gist.github.com/twitchyliquid64/72a0ef3d96ad12a61d3f82dfd4367b5e

Should work on all debian-based systems.

**Comment by paulalting** [comment]
Yes, I concur, it makes for a better install if build instructions are explicit as @thehans mentions, rather than leaving it up to users to figure it out.

Once I found this post here, I was easily able to get the install correct and things working as they should.
